### ISSUE TITLE
[Feature request] Add easier way to add/clone existing instrument to track

### ISSUE THREAD

**Issue by akimaze** [kmatheussen/radium#1395]
When the project grows, currently when you want add existing instrument to track you need find it in very long menu e.g.:

![image](https://user-images.githubusercontent.com/63190361/211163592-0541521f-bba7-4928-9859-32b4d2fb82c3.png)

So it would be great to have here a window with searching like in "From plugin manager" to simply search by instrument name/type. It can be "Search..." menu item under the "All" menu item.

Another place to speed up this task is modular mixer. It would be great to have "Assign to current track" in instrument context menu.

**Comment by kmatheussen** [comment]
Yes, the "Instrument list" window can probably be modified to be used for this (in the "Windows" meny). It sounds exactly like the thing you're describing.

**Comment by kmatheussen** [comment]
I've added a "Assign to current editor track" option to the instrument popup menu. You can assign keybinding to it as well. https://github.com/kmatheussen/radium/commit/abe9c8efc41ef75c45a16f2321836120d8805340

**Comment by akimaze** [comment]
> I've added a "Assign to current editor track" option to the instrument popup menu. You can assign keybinding to it as well. [abe9c8e](https://github.com/kmatheussen/radium/commit/abe9c8efc41ef75c45a16f2321836120d8805340)

Works great thanks :)

**Comment by luzpaz** [comment]
Is this ticket then solved ?

**Comment by akimaze** [comment]
@luzpaz I think it's partially solved/fixed. You can assign instrument to current track from mixer (and that fixes most use cases for me) but there is no window with searching. I think it's why it is still open.
